ygen/goelements.go

```diff
@@ -331,22 +331,22 @@ func (s *genState) yangTypeToGoType(args resolveTypeArgs, compressOCPaths bool)
 //	}
 //
 // Is returned with a goType of Bar_Foo_Union (where Bar_Foo is the schema
-// path to an element). The unionTypes are specified to be string and int8.
+// path to an element). The UnionTypes are specified to be string and int8.
 //
 // The compressOCPaths argument specifies whether OpenConfig path compression
 // is enabled such that the name of enumerated types can be calculated correctly.
 //
 // goUnionType returns an error if mapping is not possible.
 func (s *genState) goUnionType(args resolveTypeArgs, compressOCPaths bool) (*MappedType, error) {
 	var errs []error
-	unionTypes := make(map[string]int)
+	UnionTypes := make(map[string]int)
```

Comment:
It looks like there's a slightly over-aggressive regexp fix applied here -- within this function we shouldn't need to rename this `UnionTypes` -- i.e., the capitalisation here doesn't make a difference to the code.